Makefile

```diff
@@ -1,17 +1,40 @@
+
+ verbose ?= 5
+
 .PHONY: fmt				
 fmt:
 	@gci overwrite ./pkg/
 	@go fmt ./pkg/...
 	@gofumpt -w ./pkg/
 
-.PHONY: e2e
-e2e: fmt
-	@./cicd-scripts/run-e2e-tests.sh
+.PHONY: check
+check: fmt
+	@./cicd-scripts/check-variables.sh
+
+.PHONY: connection
+connection: fmt check
+	@./cicd-scripts/run-tests.sh -f "connection" -v $(verbose)
+
+.PHONY: label
+label: fmt check
+	@./cicd-scripts/run-tests.sh -f "label" -v $(verbose)
+
+.PHONY: policy
+policy: fmt check
+	@./cicd-scripts/run-tests.sh -f "policy" -v $(verbose)
+
+.PHONY: app 
+app: fmt check
+	@./cicd-scripts/run-tests.sh -f "app" -v $(verbose)
```

Comment:
nit: I'm thinking if there is simpler way for these make targets, given that they have similar pattern, can we just reference the build target with `$@`